### PR TITLE
Resume Generator Form

### PR DESCRIPTION
**Pull request by Gremlinbaby**
Please accept, Lawrence Approves 🥇

### CODE REVIEW

**Comment by qsaml** [comment]
might need to rebase, travis error explains better than I

**Comment by Gremlinbaby** [comment]
> might need to rebase, travis error explains better than I

I believe it is lint formatting checks.

**Comment by qsaml** [comment]
> > might need to rebase, travis error explains better than I
> 
> I believe it is lint formatting checks.

image
not sure about this, getting travis to even consider running lint means something to do with the ss here

**Comment by Gremlinbaby** [comment]
> > > might need to rebase, travis error explains better than I
> > 
> > 
> > I believe it is lint formatting checks.
> 
> image
> not sure about this, getting travis to even consider running lint means something to do with the ss here

Thank you for the suggestion!

File: src/Routing.js [hunk 1]
```diff
@@ -10,14 +10,15 @@ import EventManager from './Pages/EventManager/EventManager';
...
-
+import Resume from './Pages/Resume/resume.js';
 import Home from './Pages/Home/Home.js';
 import NotFoundPage from './Pages/NotFoundPage/NotFoundPage';
 import EventList from './Pages/Events/EventList';
 import PrintingSolids from './Pages/3DPrinting/3DPrintForm.js';
 import SolidsConsole from './Pages/3DPrintingConsole/3DConsole.js';
-import MembershipApplication from
-  './Pages/MembershipApplication/membershipApplication.jsx';
+import {
```
[the remark this answers is not in the document]
Comment (IsitaB) [reply src/Routing.js:19]: After you fix that, you'll see a few warnings that you'll have to deal with.
Screen Shot 2020-08-13 at 11 23 38 PM

[the remark this answers is not in the document]
Comment (IsitaB) [reply src/Routing.js:19]: and in the console 
Screen Shot 2020-08-13 at 11 30 09 PM

**Pull request closed without merging**